go.mod

```diff
@@ -2,53 +2,22 @@ module github.com/rh-jmc-team/container-jfr-operator
 
 require (
 	github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
-	github.com/jetstack/cert-manager v1.0.2
+	github.com/jetstack/cert-manager v1.1.0
```

Comment:
We should update the Makefile's `CERT_MANAGER_VERSION` too

Reply:
Ah yes, fix committed